schema/jhub-apps-integration.json

```diff
@@ -0,0 +1,34 @@
+{
+    "jupyter.lab.menus": {
+      "main": [
+        {
+          "id": "jp-mainmenu-deploy-app",
+          "label": "Services",
+          "disabled": false,
+          "rank": 80,
+          "items": [
+            {
+              "command": "jhub-apps:deploy-app"
+            }
+          ]
+        }
+      ],
+      "context": [
+      {
+        "command": "jhub-apps:deploy-app",
+        "selector": ".jp-DirListing-item",
```

Comment:
Question: Does `selector` really need to be here (it seems so), and if so, what should I set it as?

Reply:
Yes. I think that for now it only makes sense for files not for directories so the selector should reflect that.